azure-sdk-tools/setup.py

```diff
@@ -17,7 +17,6 @@
     'pytest>=3.5.1',
     # 'azure-devtools>=0.4.1' override by packaging needs
     'readme_renderer',
-    'azure-storage-blob',
     'azure-storage-file<2.0',
```

Comment:
I assume we removed blob because it wasn't being used in the tools? I suppose we don't have the same convenience for files?

Reply:
I haven't checked files yet. I am leaving file due to the fact that I simply don't know when `azure-storage-file` comes in. Once it DOES I need to adjust the tooling to leverage that.